docs/running-workflows/supported-systems/yadage/index.md

```diff
@@ -1,3 +1,23 @@
 # Yadage
 
-The Yadage workflow system originated in particle physics community.
+The Yadage workflow system has been developed in the context of High Energy
+Physics analyses in order to realized re-usable analyses for systematic
+reinterpretation of data collected by the Large Hadron Collider. The engine
+orchestrates the execution of a directed acyclic graph of containerized
+workloads.
+
+For additional information, please see:
+
+- Yadage documentation: [https://yadage.readthedocs.io](https://yadage.readthedocs.io)
+- Yadage tutorial: [https://yadage.github.io/tutorial](https://yadage.github.io/tutorial/)
+
+## Supported versions
+
+| REANA version         | Yadage    | Packtivity
```

Comment:
If `Packtivity` is relevant here, perhaps Adage as well?

Reply:
I'm not sure, @tiborsimko, what do you think?

Reply:
Yes, it would be good to present `(yadage, adage, packtivity)` triad.  

We could actually show also `yadage-schemas` to make it a quad, but that might perhaps be unnecessary? (Could look like an overkill: yadage/packtivity should be mostly enough)